### PR TITLE
Add video in use case landing pages (phase 1 for use case landing page)

### PR DESCRIPTION
**Pull request by ge0ffrey**

### CODE REVIEW

File: templates/useCaseBase.ftl [hunk 1]
```diff
@@ -7,6 +7,17 @@
     <@parent.layout>
         <div class="row">
             <div class="col-md-9">
+                <div class="float-start me-4 mb-4">
+                    <@macros.videoCard youtubeId=content.use_case_demo_youtube_id long=false/>
```
Comment (yurloc) [inline templates/useCaseBase.ftl:11]: The long=false card is broken. It doesn't launch the video when clicked.

**Review by yurloc — CHANGES_REQUESTED** [review]
Short card is broken.

**Comment by ge0ffrey** [comment]
@yurloc that should fix it, but it's really a pity I had to duplicate code between short and long (due to need for a "stretched-link" hover over behavior for the long card)

File: content/learn/useCases/conferenceScheduling.adoc [hunk 2]
```diff
@@ -3,7 +3,10 @@
 :jbake-description: Use OptaPlanner (Java™, open source) to optimize assigning conference talks to timeslots and rooms.
 :jbake-priority: 1.0
 :jbake-related_tag: conference scheduling
-:showtitle:
+:jbake-use_case_demo_youtube_id: ykF8DjxhQJI
+
+Conference scheduling assigns each talk to a timeslot and a room.
+Our implementation scheduled talks for for major IT events, such as Devoxx, Red Hat Summit and Voxxed.
```
Comment (yurloc) [inline content/learn/useCases/conferenceScheduling.adoc:9]:
```suggestion
Our implementation scheduled talks for major IT events, such as Devoxx, Red Hat Summit and Voxxed.
```

**Review by yurloc — APPROVED** [review]
It works now. LGTM (just a small typo found).

**Pull request merged**